src/components/SeriesSearch/SeriesSearch.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import _ from 'lodash';
+import { getEpisodes } from '../../api/api';
+import { Chart } from '../Chart/Chart';
+
+const SeriesSearch = (props) => {
+  const [episodes, setEpisodes] = useState('');
+  const [searchValue, setSearchValue] = useState('');
+
+  const handleSearchInputChange = (event) => {
+    setSearchValue(event.target.value);
+  };
+
+  const resetInputValue = () => {
+    setSearchValue('');
+  };
+
+  const generateSeasonLabels = (data) => {
+    let seasonLabels = [];
+    let seasonCount = 1;
+    _.each(data, (episodeData) => {
+      if (episodeData.episode === 1 && episodeData.season === seasonCount) {
+        const label = {
+          type: 'seasonLabels',
+          season: seasonCount,
+          episode: 0,
+        };
+        seasonLabels.push(label);
+        seasonCount++;
+      }
+    });
+    return seasonLabels;
+  };
+
+  const generateEpisodeLabels = (data) => {
+    let episodeLabels = [];
+    let maxEpisodeCount = 0;
+    _.each(data, (episodeData) => {
+      if (episodeData.episode > maxEpisodeCount) {
+        const label = {
+          type: 'episodeLabels',
+          episode: episodeData.episode,
+          id: episodeData.id,
+          title: episodeData.title,
+          season: 999, // placeholder number so episode label numbers are always on the bottom
+        };
+        episodeLabels.push(label);
+        maxEpisodeCount++;
+      }
+    });
+    return episodeLabels;
+  };
+
+  const searchFunction = async (event) => {
+    event.preventDefault();
+    setEpisodes([]);
+
+    getEpisodes(searchValue).then(({ data: { results } }) => {
+      const seasonLabels = generateSeasonLabels(results);
+      const episodeLabels = generateEpisodeLabels(results);
+      setEpisodes([...results, ...seasonLabels, ...episodeLabels]);
+    });
+
+    resetInputValue();
+  };
+
+  const seriesData = episodes;
+
+  return (
+    <div className="chart-container">
```

Comment:
this className chart-container doesnt semantically relate to its children (form + chart). I dont think we need this right now